Sources/Plasma/PubUtilLib/plDrawable/plDynaFootMgr.cpp

```diff
@@ -81,10 +79,9 @@ int plDynaFootMgr::INewDecal()
 
 plDynaFootMgr::plDynaFootMgr()
 {
-    fPartIDs.SetCount(kNumPrintIDs);
-    int i;
-    for( i = 0; i < kNumPrintIDs; i++ )
-        fPartIDs[i] = kPrintIDs[i];
+    fPartIDs.SetCount(kFootPrintIDs.size());
+    for (size_t i = 0; i < kFootPrintIDs.size(); i++)
```

Comment:
```suggestion
    fPartIDs.SetCount(std::size(kFootPrintIDs));
    for (size_t i = 0; i < std::size(kFootPrintIDs); i++)
```
(So the hard-coded size isn't needed in `std::array`)

And yes, this will eventually become simpler when everything is converted to `std::vector` >.>

Reply:
I figured you'd be able to make use of a constexpr array when vector-izing all of this ^_^